include/operations/blas3_trees.h

```diff
@@ -42,7 +42,12 @@ enum class gemm_memory_t : int { local = 0, no_local = 1 };
  * It can be either naive to use a naive algorithm, classic for the default
  * algorithms, or tall_skinny for tall and skinny matrices
  */
-enum class gemm_algorithm_t : int { naive = 0, classic = 1, tall_skinny = 2 };
+enum class gemm_algorithm_t : int {
+  naive = 0,
+  classic = 1,
+  tall_skinny = 2,
+  alternative = 3
+};
```

Comment:
"classic", "alternative"? Could we have clearer names?

Reply:
Sure. Do you have something in mind?

Reply:
Discussed offline with Mehdi and Adam. `alternative` was swallowed by `tall_skinny` and `classic` renamed to `standard` to avoid confusion with the `naive` algorithm which is sometimes called classic in the literature